packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartAxisSize.selectors.ts

```diff
@@ -6,28 +7,48 @@ import { createSelector } from '../../utils/selectors';
 
 export const selectorChartLeftAxisSize = createSelector([selectorChartRawYAxis], (yAxis) =>
   (yAxis ?? []).reduce(
-    (acc, axis) => (axis.position === 'left' ? acc + (axis.width || 0) : acc),
+    (acc, axis) =>
+      axis.position === 'left'
+        ? acc +
+          (axis.width || 0) +
+          (axis.zoom?.slider?.enabled ? (axis.zoom?.slider?.size ?? DEFAULT_ZOOM_SLIDER_SIZE) : 0)
```

Comment:
Same for other selectors
```suggestion
          (axis.zoom?.slider.enabled ? axis.zoom?.slider.size : 0)
```